components/expression_language.rst

```diff
@@ -120,8 +122,8 @@ This is how you can use these flags::
 
     $expressionLanguage = new ExpressionLanguage();
 
-    // this returns true because the unknown variables and functions are ignored
-    var_dump($expressionLanguage->lint('unknown_var + unknown_function()', Parser::IGNORE_UNKNOWN_VARIABLES | Parser::IGNORE_UNKNOWN_FUNCTIONS));
+    // this returns null because the unknown variables and functions are ignored, but don't throw `SyntaxError` exception
+    var_dump($expressionLanguage->lint('unknown_var + unknown_function()', [], Parser::IGNORE_UNKNOWN_VARIABLES | Parser::IGNORE_UNKNOWN_FUNCTIONS));
```

Comment:
I would avoid using `var_dump` here as it does not make sense.

Reply:
I agree, I dont remove it to limit the changes.